client/src/constants/form-options.js

```diff
@@ -9,3 +9,141 @@ export const YesNoPreferNot = [
 ];
 
 export const YesNoDontKnow = [...YesNo, { value: `I don't know`, label: `I don't know` }];
+
+export const currentOrMostRecentIndustryOptions = [
```

Comment:
I found that in some cases, we define values as an array and map options dynamically if the value and label are the same. See ArchiveHiredParticipantForm.formatOptions.